docs/product-user-guide/src/main/asciidoc/business-central-con.adoc

```diff
@@ -16,15 +16,15 @@ Business Central integrates the following tools:
 endif::BA[]
 
 ifdef::DM[]
-Business Central is the web-based user interface for both {PRODUCT_DM} 6 and {PRODUCT_BA} 6. Business Central allows business analysts, rule experts, developers, and rule administrators to manage rules in a multiuser environment and implement changes in a controlled fashion. Business Central has been combined with the core drools engine and other tools for optimal performance.
+{CENTRAL} is the web-based user interface for both {PRODUCT_DM} 6 and {PRODUCT_BA} 6. {CENTRAL} allows business analysts, rule experts, developers, and rule administrators to manage rules in a multiuser environment and implement changes in a controlled fashion. {CENTRAL} has been combined with the core drools engine and other tools for optimal performance.
```

Comment:
Should the "6" become a variable for the actual version number? (I assume it should be "7" in these documents and could obviously change again in the future).

Reply:
@michelehaglund